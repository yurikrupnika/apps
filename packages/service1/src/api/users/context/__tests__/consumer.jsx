import React from 'react';
import Consumer from '../consumer';

const {
    describe,
    test,
    expect,
    render,
    shallow,
    mount
} = global;

describe('currency consumer', () => {
    test('render consumer with render props', () => {
        const wrapper = mount( // notice render here
            <Consumer render={(props) => {
                const { fetch } = props;
                fetch();
                return (
                    <div>
                        asd
                    </div>
                );
            }}
            />
        );
        expect(wrapper).toMatchSnapshot();
    });
});
