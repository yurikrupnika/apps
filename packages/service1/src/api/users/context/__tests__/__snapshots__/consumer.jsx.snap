// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`currency consumer render consumer with render props 1`] = `
<UsersConsumer
  render={[Function]}
>
  <DataContainer
    data={Array []}
    fetch={[Function]}
    loading={false}
    render={[Function]}
    selected={null}
  >
    <div>
      asd
    </div>
  </DataContainer>
</UsersConsumer>
`;
